sdk/metricsadvisor/tests.yml

```diff
@@ -0,0 +1,33 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      BuildTargetingString: azure-ai-metricsadvisor
+      ServiceDirectory: metricsadvisor
+      EnvVars:
+        AZURE_SUBSCRIPTION_ID: $(provisioner-subscription)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_METRICS_ADVISOR_ENDPOINT: $(metricsadvisor-test-service-endpoint)
+        AZURE_METRICS_ADVISOR_SUBSCRIPTION_KEY: $(metricsadvisor-test-subscription-key)
```

Comment:
Do we share these variables with other language?

Reply:
Yes, all environment variables are shared across languages except API key